packages/moonstone/Panels/Header.js

```diff
@@ -17,6 +17,9 @@ const MarqueeH2 = MarqueeDecorator('h2');
 
 const CompactTitleBase = kind({
 	name: 'CompactTitle',
+	styles: {
+		className: 'compactTitle'
```

Comment:
This probably needs the `css` reference. Otherwise, it'll apply a global `compactTitle` class rather than the modularized version